### PR TITLE
Allow minReplicas to be 0 for autoscaling

### PR DESCRIPTION
**Pull request by KevinFCormier**
Enforce maxReplicas to be at least 1 per machine set

https://issues.redhat.com/browse/ACM-7812

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -1701,6 +1701,7 @@
   "machinePool.modal.scale.maxReplicas.label": "Maximum replicas",
   "machinePool.modal.scale.minReplicas.label": "Minimum replicas",
   "machinePool.modal.scale.replicas.label": "Machine set replica count",
+  "machinePool.modal.scale.validation.maxReplicas": "Maximum replicas must be greater than or equal to to the number of machine sets ({{machineSetCount}}).",
```
Comment (oafischer) [inline frontend/public/locales/en/translation.json:1704]:
```suggestion
  "machinePool.modal.scale.validation.maxReplicas": "Maximum replicas must be greater than or equal to the number of machine sets ({{machineSetCount}}).",
```

**Comment by oafischer** [comment]
Just one typo, otherwise looks good @KevinFCormier. I only made one suggestion but the typo needs to be fixed in all instances.

File: frontend/public/locales/en/translation.json (hunk 1 above)
Comment (KevinFCormier) [reply frontend/public/locales/en/translation.json]: Thank you!

**Review by oafischer — APPROVED** [review]

**Comment by KevinFCormier** [comment]
Oops... I didn't realize the approval from Oliver would let it merge. @Randy424 can you still review please, and let me know if you think any changes are needed? Thanks.

**Review by Randy424 — COMMENTED** [review]
/lgtm

**Comment by Randy424** [comment]
Looks fine.

**Pull request merged**